### ISSUE TITLE
TypeError: Object ’ has no method 'equals'

### ISSUE THREAD

**Issue by mbrevda** [substack/quote-stream#3]
Since approximatly https://github.com/substack/quote-stream/commit/d628f907e0e7e88e040e4cd3f818f9b60943ade5, I have been geting the following error:

```
/pipeline/build/node_modules/brfs/node_modules/quote-stream/index.js:70
                if (rawBuf.equals(multiByteBuffers.lineSeparator)) {
                           ^
TypeError: Object ’ has no method 'equals'
    at DestroyableTransform.write [as _transform] (/pipeline/build/node_modules/brfs/node_modules/quote-stream/index.js:70:28)
    at DestroyableTransform.Transform._read (/pipeline/build/node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:10)
    at DestroyableTransform.Transform._write (/pipeline/build/node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:160:12)
    at doWrite (/pipeline/build/node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:326:12)
    at writeOrBuffer (/pipeline/build/node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:312:5)
    at DestroyableTransform.Writable.write (/pipeline/build/node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:239:11)
    at write (_stream_readable.js:602:24)
    at flow (_stream_readable.js:611:7)
    at ReadStream.pipeOnReadable (_stream_readable.js:643:5)
    at ReadStream.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at emitReadable (_stream_readable.js:423:5)
    at readableAddChunk (_stream_readable.js:166:9)
    at ReadStream.Readable.push (_stream_readable.js:128:10)
    at onread (fs.js:1575:12)
    at Object.wrapper [as oncomplete] (fs.js:466:17)
```

Seems the apostrophe in this string is the culprit: `player’s`

**Comment by Mortimerp9** [comment]
getting the same error here with a different string:
```
....node_modules/brfs/node_modules/quote-stream/index.js:70
                if (rawBuf.equals(multiByteBuffers.lineSeparator)) {
                           ^
TypeError: Object ╳ has no method 'equals'
    at DestroyableTransform.write [as _transform] (....node_modules/brfs/node_modules/quote-stream/index.js:70:28)
    at DestroyableTransform.Transform._read (....node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:10)
    at DestroyableTransform.Transform._write (....node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:160:12)
    at doWrite (....node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:326:12)
    at writeOrBuffer (....node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:312:5)
    at DestroyableTransform.Writable.write (....node_modules/brfs/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:239:11)
    at write (_stream_readable.js:602:24)
    at flow (_stream_readable.js:611:7)
    at ReadStream.pipeOnReadable (_stream_readable.js:643:5)
    at ReadStream.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at emitReadable (_stream_readable.js:423:5)
    at readableAddChunk (_stream_readable.js:166:9)
    at ReadStream.Readable.push (_stream_readable.js:128:10)
    at onread (fs.js:1575:12)
    at callback (....node_modules/qfnode/node_modules/watchify/node_modules/chokidar/node_modules/readdirp/node_modules/graceful-fs/polyfills.js:207:17)
    at Object.wrapper [as oncomplete] (fs.js:466:17)
```

did an clean update of my npm packages this morning.

**Comment by Mortimerp9** [comment]
seems to be coming for a brfs sync read to do somehting like this:
```
var template = fs.readFileSync(__dirname + '/../ractive/explore-matrix-timeseries.html', 'utf8');
```

when the file contains some utf8 character (tried with ╳, ○)

**Comment by NHQ** [comment]
I am fairly certain this is a node version issue, as ```rawBuf``` is a node buffer.  I am running v0.10.xx and there is no ```Buffer.equals``` method.  

Solution is revert to brfs 1.4.0 or upgrade node.

**Comment by substack** [comment]
fixed in 1.0.2

**Comment by mbrevda** [comment]
Thanks!
